roles/openstack-setup/tasks/cinder.yml

```diff
@@ -19,6 +19,15 @@
     - (item.name in ['cinder','cinderv2','cinderv3'])
   run_once: true
 
+# FIXME: this task should be removed once old service is disabled for all envs
```

Comment:
If there is no api we should make mysql call but leaving service disabled will lead to confusion. The service is retired and should be removed. Here is link to sample, note sql query will need to be adjusted.
https://github.com/blueboxgroup/ursula/blob/3430bd6e331ff14d09e6db4fc1147a517e78abf9/playbooks/remove_controller_services/tasks/main.yml#L65